code/game/gamemodes/endgame/scoreboard/department_score.dm

```diff
@@ -76,19 +76,16 @@
 
 /datum/controller/gameticker/scoreboard/proc/service_score()
 	//Janitor
-	//Check how many uncleaned mess are on the station. We can't run through cleanable for reasons, so yeah, long
+	//Check how many uncleaned mess are on the station.
+	var/list/valid_areas = the_station_areas.Copy() - typesof(/area/holodeck) //excluding these for now because subtypes are on z2 normally
```

Comment:
Does this mean walking into the holodeck counts as being off-station?

Reply:
for now, until i can verify /area/holodeck/alphadeck doesn't get overwritten when the holodeckchanges